frontend/src/modules/entitydetails/components/ContextIssueButton.css

```diff
@@ -0,0 +1,24 @@
+.metadata .source-string-comment {
+    flex: 1 0 auto;
+}
+
+.metadata .source-string-comment .context-issue-button {
+    background: #333941;
+    border: 1px solid #3f4752;
+    border-radius: 4px;
+    color: #ccc;
+    float: right;
+    font-size: 11px;
+    font-style: normal;
+    font-weight: 100;
+    margin: -1px -1px 0 4px; /* Compensate for invisible border in default state */
```

Comment:
I noticed that `-1px -1px -1px 4px` allows the 2nd line of the source string to occupy full width.